phys/module_diag_hailcast.F

```diff
@@ -209,10 +210,20 @@ SUBROUTINE hailcast_diagnostic_driver (   grid , config_flags     &
         ( (.not. grid%nested) .or. &
         ( (grid%nested) .and. (abs(grid%dtbc) < 0.0001) ) ) )THEN
        doing_adapt_dt = .TRUE.
-       IF ( haildtacttime .eq. 0. ) THEN
-          haildtacttime = CURR_SECS + haildt
-       END IF
-    END IF
+       !IF ( haildtacttime .eq. 0. ) THEN
```

Comment:
I believe this applies broadly throughout the edits -
If code is no longer applicable please remove it rather than commenting it out

Reply:
Removed.

Reply:
Removed commmented-out/buggy code throughout